src/redux/categories/categorySlice.js

```diff
@@ -0,0 +1,15 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  amount: 0,
+  isLoading: true,
+};
+
+const categorySlice = createSlice({
+  name: 'category',
+  initialState,
+  reducers: {},
```

Comment:
Nice job so far😉. Just as required for this milestone, we ought to create a reducer that checks the status and always returns "Under construction" 😉. Kindly do well to implement this in the reducers👍🏼.

<img width="1019" alt="image" src="https://github.com/khanh-devos/rjs-bookstore-khanh/assets/64643995/4686aa5e-9887-4a4b-af3e-8a01288c107f">